src/test/java/picard/fingerprint/CheckFingerprintTest.java

```diff
@@ -56,10 +56,10 @@ private void rmTemp() {
     @Test
     public void testBaseOutput() {
         String[] args = new String[]{
-                "I=" + TEST_INPUT_VCF1,
+                "I=" + TEST_INPUT_VCF1.getAbsolutePath(),
```

Comment:
Why is this necessary? Why don't you move the `getAbsolutePath()` to where `TEST_INPUT_VCF1` is declared?

Reply:
will do.